testing/iframe.js

```diff
@@ -82,6 +82,11 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
       // Flag as being a test window.
       win.AMP_TEST_IFRAME = true;
       win.AMP_TEST = true;
+      // Set the testLocation on iframe to parent's location since location of
```

Comment:
Please do the same in the `createIframePromise` right after `iframe.contentWindow.AMP_TEST_IFRAME = true;`

And if possible, please also do this in RealWinFixture, right after `win.AMP_TEST_IFRAME = true;`.